css/style.css

```diff
@@ -0,0 +1,269 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  .main {
+    margin: auto;
+    width: 1440px;
```

Comment:
Use width: 100%; in here  instead of a fixed width.

Reply:
Thanks.